html/browsers/windows/browsing-context-names/browsing-context-default-name.html

```diff
@@ -5,22 +5,25 @@
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
 <div id="log"></div>
-<iframe src="message.html" style="display:none"></iframe>
```

Comment:
Correct. After I get through the (lots of) tests to add/edit in this section I'll end with a final cleanup patch that'll get some of this stuff more clearly sorted out.